src/interfaces/protocols/handlers/protocols-configure.ts

```diff
@@ -62,7 +58,9 @@ export class ProtocolsConfigureHandler implements MethodHandler {
         author,
         ... message.descriptor
       };
-      await StorageController.put(this.messageStore, this.dataStore, tenant, incomingMessage, indexes, dataStream);
+      // FIXME: indexes, Property 'dataSize' is incompatible with index signature.
```

Comment:
Hmm `indexes` ultimately gets passed down to `IndexLevel#put` as an `Entry` which is very general.
```typescript
type Entry = {
  [property: string]: unknown
};
```
There's a couple ways we could go with this.

@dcrousso What's your recommendation here? Seems like an actual issue if someone loads their own message store and we pass them an object with an unexpected shape.

Reply:
IMO we should allow for anything as the value.  i dont think it's super flexible for `MessageStore` to require that the `indexes` be a flat structure.  that's just how we do it :)